### ISSUE TITLE
Add ability to Auto-Fill contact information

### ISSUE THREAD

**Issue by kylehickinson** [brave/brave-ios#3390]

**Comment by iceguy154** [comment]
This feature would be KEY. Should be high priority

**Comment by mrpineapples** [comment]
I agree with @iceguy154 this should be a high priority, other ios browsers have this feature 🙏🏼

**Comment by aehlke** [comment]
Any roadmap update on this?

**Comment by feldlef** [comment]
Craving this aswell.. it's almost a standard feature for browsers by now :0 

Anyone seen it on a roadmap?

**Comment by aehlke** [comment]
firefox-ios is currently adding it
